presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/compose/BottomNavigation.kt

```diff
@@ -19,7 +20,11 @@ fun BottomNavigation(
         .fillMaxWidth()) {
         WhiteBackButton(onClick = onBackClicked, enabled = backEnabled)
         Spacer(modifier = Modifier.weight(1f))
-        BlackNextButton(onClick = onNextClicked, enabled = nextEnabled)
+        if (nextText != null) {
+            BlackButton(onClick = onNextClicked, text = nextText)
```

Comment:
FYI: Since there is talk about using "Submit" for the last survey question, this needs to support whether or not it's enabled.

Reply:
Fixed.